### PR TITLE
Build script updates

### PR DESCRIPTION
**Pull request by HayoBaan**
* Added local (optimized) build target
* Added message on code signing requirement for Mojave

### CODE REVIEW

File: aquamacs/build/build.sh [hunk 1]
```diff
@@ -97,5 +109,12 @@ test $OMIT_SYMB || dsymutil src/emacs
 
 echo ${FINALMESSAGE}
 echo "Build finished."
+echo "IMPORTANT:"
+echo "When building for Mac OS X Mojave (10.14) and later,"
+echo "please make sure you sign the executable."
+echo "  codesign -s \"<certificate>\" --deep /nextstep/Aquamacs.app"
```
Comment (davidswelt) [inline aquamacs/build/build.sh]: Did you see the "sign" script?

**Review by davidswelt — CHANGES_REQUESTED** [review]
point to sign script instead?

**Comment by HayoBaan** [comment]
Hi David, the problem with the script is that it has your certificate hard-coded and apart from you, others won't have this so the script won't work. Also your script must be run from the Aquamacs.app directory so it can't simply be called directly.

We could change that script a little bit so instead of your hard-coded certificate, it uses the one as set by the environment variable CERT and switches to the right directory. I suggest we cleanup the sign script too and remove everything that is commented-out.

However, perhaps an even better way would be this:
If the user has set CERT we automatically run the signing in the build script and if it is not set we emit the warning I put in?

**Comment by HayoBaan** [comment]
Instead of CERT perhaps AQUAMACS_CERT is a safer variable choice?

**Comment by davidswelt** [comment]
OK.  Your solution is good, although we can also just leave your change as is and rename the sign script to say "sign-release".
By the way, the real problem is the event handling.  That needs to be fixed.

**Comment by HayoBaan** [comment]
I made some changes and now the build script automatically signs the code with the certificate in AQUAMACS_CERT. Also renamed the sign script to sign-release as per your suggestion.

**Comment by HayoBaan** [comment]
> By the way, the real problem is the event handling. That needs to be fixed.

Yes, that's true; Aquamacs should not need to be able to “Control the Computer”. However since this problem is also in the “original” Mac emacs port, it's not something Aquamacs introduced so we are probably better off leaving it to the original code maintainers to solve.

**Pull request merged**